### ISSUE TITLE
Font not accurate

### ISSUE THREAD

**Issue by Forceh91** [Forceh91/retro-env-can-weather-chan#1]
I'm using a placeholder font now whilst I try and find an accurate replacement. I'm suspecting there won't be an exact match for it unfortunately.

**Comment by Axon07** [comment]
Rysse is close but not the same
https://www.whatfontis.com/NMY_Rysse.font

**Comment by Forceh91** [comment]
What I came to realize during my hunt for a font was that I think this was probably generated by the hardware it was running on and not really a "font" as such.

I think the only option for a truly accurate font would be to painstakingly recreate it from scratch using source material

**Comment by wpgne** [comment]
Do you happen to know what pixel or measurement each character should be if I look at creating them from scratch?

**Comment by Forceh91** [comment]
I'm actually not sure on that, you'll probably have to measure them manually from source material (played back at the resolution it was recorded in). I know the font is monospaced though; all the characters have the same width/height.

**Comment by wpgne** [comment]
But, what would be ideal for displaying on your channel/app? Is it the same resolution as a standard definition broadcast?

**Comment by Forceh91** [comment]
The broadcast is SD yeah, 640x480. I'm not 100% how font files actually work but I can set the size to whatever I want when I display it in the channel.

**Comment by wpgne** [comment]
How does this font look for the primary text? I know it's not exact, but could work until we come up with something else. I really would like to see those crossed zeros removed.
[Syne_Mono.zip](https://github.com/Forceh91/retro-env-can-weather-chan/files/10051856/Syne_Mono.zip)

**Comment by Forceh91** [comment]
It looks.. weird to me. I think it's the way the letter stroke randomly changes width at points? I should think we can find a monospaced "console style" font without any crossed zeros if that's what we start out with

**Comment by Forceh91** [comment]
@wpgne Is this _the_ font? It looks so so so close to me: [VT323](https://fonts.google.com/specimen/VT323?category=Monospace&preview.text=A%20B%20C%20D%20E%20F%20G%20H%20I%20J%20K%20L%20M%20N%20O%20P%20Q%20R%20S%20T%20U%20V%20W%20X%20Y%20Z%200123456789&preview.text_type=custom)

**Comment by wpgne** [comment]
VT323 isn't bad -- no crossed zeros. But the I is just a line and the Q's tail is up into the letter on the genuine font. If it looks good to you on your end, this seems like a worthy candidate.

**Comment by wpgne** [comment]
It looks pretty decent - and you've made the text size larger too -- filling up the space.

**Comment by Forceh91** [comment]
Yep from what I can tell from source material the font size was around 30-32px (high) so I've set it to 30px.

**Comment by wpgne** [comment]
I tried looking for the second font needed (for the scroll and ECW title at the bottom). The one here: https://user-images.githubusercontent.com/117599578/202945001-942c0f84-b878-4aca-b703-53204f7f2330.png is not bad, but the I isn't monospaced. 
The key with this second font is that all letters are in uppercase, but capitals are taller than the "lowercase" ones. There doesn't seem to be many fonts around like that.

**Comment by Forceh91** [comment]
Yeah that's a weird one. I suppose one way around that would be to just have a font where the uppercase are all the same size and make the E C and W a slightly larger font size.

**Comment by Forceh91** [comment]
This has been put onto the live channel and will merge the PR if everything looks good. I've made it a config option so that anyone running the channel can pick between VT323 (default) and VCR.

**Comment by wpgne** [comment]
Can you do that with the taller capital/number/symbols via scripting with any font then (for the secondary font)?
Just checked it out ... looks pretty good. A marked improvement with the crossed zeros gone and larger text. Would be ideal to have it fill up the space completely and align the time/date accordingly in the future.
I was thinking too -- if we could get a font like the Videon unit had, you could possibly config it to also display that way.
One thing at a time though lol

**Comment by Forceh91** [comment]
Yeah in theory. It's not an ideal solution but it does work, lol.

![image](https://user-images.githubusercontent.com/14955663/204117396-029d9990-cfa3-4ca3-9705-530929063269.png)

**Comment by Forceh91** [comment]
The font stuff has been merged to master now but I'll keep this open whilst we figure out the few last details

**Comment by wpgne** [comment]
I see you got a response from VSP on Mastodon about the font

**Comment by wpgne** [comment]
Have you heard anything new on this?

**Comment by Forceh91** [comment]
We may be close to a font, I asked for further updates earlier

**Comment by wpgne** [comment]
We have @djrrb looking into this via a repository of his
